custom_components/ramses_cc/broker.py

```diff
@@ -92,15 +92,23 @@ async def start(self) -> None:
             client_state.get(SZ_SCHEMA, {}) if restore_schema else {},
         )
 
-        cached_packets = (
-            self._filter_cached_packets(
-                client_state.get(SZ_PACKETS, {}), restore_schema
-            )
-            if restore_state
-            else {}
-        )
+        @callback
+        def cached_packets() -> dict:
```

Comment:
Let's try not to use generic types if possible. At least `dict[str, Any]`, but we can have:
```python
def cached_packets() -> dict[str, str]:  # dtm_str, packet_as_str
```

Eventually, I'd be making everything strict-typed, and I've discovered it's a real pain to retro-fit... But I pick up so many bugs doing it.

Reply:
Yeah, good shout.